README.md

```diff
@@ -118,6 +99,7 @@ Compiling a HIP application with CHIP-SPV will allow you to execute HIP code on
 You can find various HIP applications here: <https://github.com/CHIP-SPV/hip-testsuite>
 
 ```bash
+export HIP_PLATFORM=spirv
```

Comment:
Should not be needed. `hipcc` looks for `$HIP_PATH/lib/.hipInfo` which defines `HIP_PLATFORM=spirv` and other important variables. 

(If HIP_PATH is not defined it defaults to <path-to-hipcc>/..).